### DISCUSSION TITLE
Persian [Farsi] language support

### DISCUSSION THREAD

**Discussion by Hadryan** [meilisearch/product#137] [Feedback & Feature Proposal]
> There are approximately 110 million Persian speakers worldwide, including Persians, Tajiks, Hazaras, Caucasian Tats and Aimaqs. (Wikipedia)

Thanks for this great project, that would be great if there will be support for that.

**Comment by Kerollmops** [comment]
Hey @Hadryan,

As I just [answered to another language support request for Russian](https://github.com/meilisearch/MeiliSearch/issues/552#issuecomment-605123984), we would like every feedback you can give, this way we would be able to know what's the state of the engine.

You can find more information about how the engine splits text in the comment I linked above :)

Thank you very much!

**Comment by Hadryan** [comment]
That would be great, thanks. Please let these language support issues open, so that other people may work on them.

Thanks again
